test/CMakeLists.txt

```diff
@@ -53,13 +53,44 @@ string(RANDOM LENGTH 9 ALPHABET 0123456789 RANDOM_SEED)
 message(STATUS "Random seed is ${RANDOM_SEED}")
 set(TEST_PARAM_DELIM ":")
 
-list(APPEND LUATEST_COMMAND_WITH_FLAGS
-    ${PROJECT_SOURCE_DIR}/test-run/lib/luatest/bin/luatest
-    --no-clean
-    --verbose
-    --no-capture
-    --output tap
-)
+function(add_luatest_test suite_name test_name shuffle)
+    set(test_title "test/${suite_name}/${test_name}")
+    set(test_log "${VARDIR}/${suite_name}-${test_name}.log")
+    set(test_env
```

Comment:
Why do you use `set()` instead `list()`?

Reply:
What's the difference? Is there any reason why I should prefer `list` over `set`?

Reply:
According to the [CMake documentation](https://cmake.org/cmake/help/latest/command/list.html), `set` can be used for creating a list:

<img width="1020" height="215" alt="image" src="https://github.com/user-attachments/assets/4c691497-6d58-488e-a430-e8dd5c22ee84" />

We usually use `set` in Tarantool source so I prefer to leave it as is. Anyway, this hunk was reverted so this comment isn't longer relevant.

Reply:
Okay.